bes_theme/assets/data/version_details/376-oneCCL-Versiondetails.json

```diff
@@ -0,0 +1,9 @@
+[
+    {
+        "version": "2021.7.1",
+        "release_date": "",
```

Comment:
@sudhirverma Please provide the release date

Reply:
done